themes_cern/indico_themes_cern/themes/rucio.scss

```diff
@@ -0,0 +1,15 @@
+@import 'base/palette';
+
+$header-bg-color: #2e3842;
+$header-icon-color: $light-black;
+$header-text-color: $light-gray;
+
+@import 'themes/indico';
+
+@include header-logo('/static/plugins/themes_cern/rucio_logo.png', 15px, 200px, 12%);
```

Comment:
@include header-logo('themes_cern:rucio_logo.png', ...);

Reply:
Oh cool. Had forgotten abou that ;)